sdk/storage/azure-storage-blob/swagger/README.md

```diff
@@ -16,7 +16,7 @@ autorest
 ### Code generation settings
 ``` yaml
 use: '@autorest/java@4.1.0'
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/e515b6251fdc21015282d2e84b85beec7c091763/specification/storage/data-plane/Microsoft.BlobStorage/preview/2021-04-10/blob.json
```

Comment:
Why were we generating off 2021-04-10 swagger before?  This is the STG 81 service version, and we have added lots of features since then.